src/constant/routes.ts

```diff
@@ -18,6 +18,10 @@ const routes = {
   },
   interviewPrep: '/interview-prep',
   interviewPrepExplore: '/interview-prep/explore',
+  allSheets: {
+    javaScriptInterviewSheet: '/interview-prep/javascript-interview-sheet',
+    javaScriptBasicsInterviewSheet: '/interview-prep/javascript-basics-interview-sheet',
```

Comment:
Currently do it for JS only. Remove second route and also from other places. Use Search in VS code and remove